pkg/scheduler/framework/plugins/helper/helpers.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package helper
+
+import "k8s.io/api/core/v1"
+
+// GetPersistentVolumeClaimClass returns StorageClassName. If no storage class was
+// requested, it returns "".
```

Comment:
Maybe add a comment that this came from `pkg/apis/core/v1/helper`. This function does seem to have a bunch of other uses through the codebase, but it's trivial enough that I don't see a problem duplicating it to remove a dependency.

Reply:
done